core/executor/src/tests/system_script/metadata.rs

```diff
@@ -207,6 +208,43 @@ fn prepare_validator() -> ValidatorExtend {
     }
 }
 
+#[test]
+fn test_set_ckb_related_info() {
+    // Init ckb info db.
+    {
+        let inner_db = RocksAdapter::new(CKB_INFO_ROCKSDB_PATH, Default::default())
+            .unwrap()
+            .inner_db();
+        let mut _db = METADATA_DB.write();
+        const METADATA_DB_CACHE_SIZE: usize = 10;
+        _db.replace(Arc::new(RocksTrieDB::new_metadata(
+            Arc::clone(&inner_db),
+            METADATA_DB_CACHE_SIZE,
+        )));
+    }
+
+    let metadata_root = H256::zero();
+    let mut store = MetadataStore::new(metadata_root).unwrap();
+    let ckb_infos = CkbRelatedInfo {
+        metadata_type_id:     H256::from_str(
+            "0xdb0782aba62896c2a7c279f3de8dbbd7fd06729cc8b7b499df93f5c450f61839",
+        )
+        .unwrap(),
+        checkpoint_type_id:   H256::zero(),
+        xudt_args:            H256::zero(),
+        stake_smt_type_id:    H256::zero(),
+        delegate_smt_type_id: H256::zero(),
+        reward_smt_type_id:   H256::zero(),
+    };
+    let _result = store.set_ckb_related_info(&ckb_infos);
+
+    CURRENT_METADATA_ROOT.with(|root| {
+        let new_root = *root.borrow();
+        println!("new_root: {:?}", new_root);
+        assert_ne!(new_root, metadata_root);
+    });
```

Comment:
The unit test does not cover the changes.

Reply:
In unit test, if set_ckb_related_info was executed, then get_ckb_related_info will get the data correctly (because the data in cache has been update).  
So I just compared the original root and updated root to check.
Any better suggestions?

Reply:
You should check the new root is different from the origin root.

Reply:
> You should check the new root is different from the origin root.

Yes, the metadata_root is the origin root.